server/controller/utils/splitString.js

```diff
@@ -0,0 +1,11 @@
+const splitString = (string) => {
+
+    // 1. Replace all "\n" inside string(the input of the function) with empty space
+    // 2. Split the string using "•"
+    // 3. Delete the first item from the resultant array 
+    const splitedString = string.replace(/\n/g,'').split('•')
```

Comment:
I see that you edit the splitedString below so you can use let instead of const

Reply:
Done